consensus/config.go

```diff
@@ -2,9 +2,20 @@ package consensus
 
 import "time"
 
+// timeout constant
 const (
-	// blockDuration is the period a node try to publish a new block if it's leader
-	blockDuration  time.Duration = 5 * time.Second
 	receiveTimeout time.Duration = 5 * time.Second
-	maxLogSize     uint32        = 1000
+	// The duration of viewChangeTimeout; when a view change is initialized with v+1
+	// timeout will be equal to viewChangeDuration; if view change failed and start v+2
+	// timeout will be 2*viewChangeDuration; timeout of view change v+n is n*viewChangeDuration
+	viewChangeDuration time.Duration = 30 * time.Second
+
+	bootstrapDuration time.Duration = 60 * time.Second
```

Comment:
what is bootstrap duration? Please add a comment.

Reply:
When network start, it takes a longer time for all the nodes to be ready. So I make the bootstrap time longer, otherwise, the nodes might start view change too soon. The exact time period might need some adjustment.